### DISCUSSION TITLE
Method Not Allowed.

### DISCUSSION THREAD

**Discussion by devbyVeriTech** [pockethost/pockethost#415] [Q&A]
Error original: {"data": {"code": 405, "data": {}, "message": "Method Not Allowed."}, "status": 405, "url": "https://fintech.pockethost.io/:8090/api/collections/users/records"}
 (NOBRIDGE) LOG  Error data: {"code": 405, "data": {}, "message": "Method Not Allowed."}

**Comment by benallfree** [comment]
Closing due to lack of information.

**Comment by devbyVeriTech** [comment]
Sorry about that.
I tried logging in my app and this is the response I got.

I was using 127.0.0.1 while developing and everything was okay. Replacing the Ip with my pockethost instance is showing this error

**Comment by benallfree** [comment]
No problem, please send the exact URL that is producing the error.
